src/components/Slider/utils.ts

```diff
@@ -88,7 +88,7 @@ function createMarks(points: number[]): RcSliderProps['marks'] {
     return marks;
 }
 
-export function getInnerState({
+export function prepareSliderInnerState({
```

Comment:
Why user might needed this function?

Reply:
> Why user might needed this function?

I need it to use in RangeSliderPicker component to get inner state. And I don't want to duplicate the code. 
Also it could be helpful for other users in the development of components based on the slider.

Reply:
@amje, I don't know is it really "fix" but not "chore"?
Is it important enough to get into the changelog?

Reply:
Yes, it's important to note this change in the changelog because you have changed the implementation

Reply:
It's just refactoring to make code more readable without any changes in implemenation